spyder/plugins/toolbar/container.py

```diff
@@ -48,17 +48,6 @@ class ToolbarActions:
     ShowToolbars = "show toolbars"
 
 
-class QActionID(QAction):

```

Comment:
Why did you remove this? We need it for some changes part of our `5.x` branch and that are going to land in 5.1.2

If possible, please restore it.

Reply:
I have restored the code for `QActionID`. But I am afraid that the other change in the file is required. See the commit `pyside: Do not modify QAction.__class__ in toolbar plugin…`.